unitTests/ui/SimpleFilterTest.ts

```diff
@@ -224,5 +228,90 @@ export function SimpleFilterTest() {
         expect($$(aSimpleFilter.cmp.root).findAll('.coveo-dropdown-background').length).toEqual(1);
       });
     });
+
+    describe('with clear button functionality', () => {
+      let simpleFilterWithEraser: Mock.IBasicComponentSetup<SimpleFilter>;
+      let simpleFilterContainer: Dom;
+
+      function clickOnClearButton() {
+        $$(simpleFilterContainer.find('.coveo-simplefilter-eraser')).trigger('click');
+      }
+
+      function selectSomeValues() {
+        simpleFilterWithEraser.cmp.selectValue('foo');
+        simpleFilterWithEraser.cmp.selectValue('bar');
+      }
+
+      function getClearButton() {
+        return $$(simpleFilterContainer.find('.coveo-simplefilter-eraser'));
+      }
+
+      beforeEach(() => {
+        simpleFilterWithEraser = Mock.optionsComponentSetup<SimpleFilter, ISimpleFilterOptions>(SimpleFilter, <ISimpleFilterOptions>{
+          ...aSimpleFilter.cmp.options,
+          enableClearButton: true
+        });
+        simpleFilterContainer = $$(simpleFilterWithEraser.cmp.element);
```

Comment:
Following on the above comment, is the `simpleFilterContainer` variable really needed? If so, what about using a function instead of needing to "maintain" a variable?

Reply:
I think that helps a bit to declutter how the test look and also a bit of performance because less function calls are required.

Reply:
Cool